src/sections/Learn/LearnPage-Sections/workshops.js

```diff
@@ -212,6 +212,7 @@ const WorkshopsSection = () => {
 `
   );
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Please remove the eslint disabling.

Reply:
Yes but if i remove this it will create a conflict for eslint because eslint wouldn't allow us to unused  variables